mob.go

```diff
@@ -467,6 +475,20 @@ func setBoolean(s *bool, key string, value string) {
 	debugInfo("Overwriting " + key + " =" + strconv.FormatBool(boolValue))
 }
 
+func setMobDoneSquash(configuration *Configuration, key string, value string) {
```

Comment:
Hm, not sure whether we really benefit from the enum here. Just using a string would probably be simpler.

Reply:
It is a string. It's just masked with its own type. This helps with refactoring. Also, it makes sure that we don't accidentally use invalid values, so it isolates this invariant. And on top of that, it helps me with coding, as I don't have to remember what exactly the string was, I just use auto-completion. I understand, that the translation code from env variable to this new Type is a little complicated. But it would be as complicated if it was a string. So if you asked me, I'm all for the enumish Type wrapping.

Reply:
I just improved on the code that parses the DoneSquash configuration. Its odd. Even though they are just strings, I cannot compare them to strings.